src/debugger/gui/CartCDFInfoWidget.cxx

```diff
@@ -23,16 +23,28 @@ CartridgeCDFInfoWidget::CartridgeCDFInfoWidget(
     int x, int y, int w, int h, CartridgeCDF& cart)
   : CartDebugWidget(boss, lfont, nfont, x, y, w, h)
 {
-  uInt16 size = 8 * 4096;
-
+  uInt32 size;
   ostringstream info;
-  info << describeCDFVersion(cart.myCDFSubtype) << " cartridge\n"
-  << "32K ROM, seven 4K banks are accessible to 2600\n"
-  << "8K CDF RAM\n"
-  << "CDF registers accessible @ $FFF0 - $FFF3\n"
-  << "Banks accessible at hotspots $FFF5 to $FFFB\n"
-  << "Startup bank = " << cart.startBank() << "\n";
 
+  if (cart.myCDFSubtype == CartridgeCDF::CDFSubtype::CDFJplus) {
+    size = 512 * 1024;
+    info << describeCDFVersion(cart.myCDFSubtype) << " cartridge\n"
+      << "512K ROM (seven 4K banks are accessible to 2600)\n"
```

Comment:
Still only 7 banks? Why 8 hotspots, then?

Reply:
The hotspot at $FFFB is being hit at startup on real hardware (not sure why), which means it must be mapped to the startup bank (or removed).    With CDFJ it is rare to need more than 1 or 2 Atari banks, so 7 should be plenty.